larky/src/main/java/com/verygood/security/larky/modules/vgs/cerebro/text/analyzer/analyzers/NoopTextPIIAnalyzer.java

```diff
@@ -0,0 +1,27 @@
+package com.verygood.security.larky.modules.vgs.cerebro.text.analyzer.analyzers;
+
+import com.verygood.security.larky.modules.vgs.cerebro.text.analyzer.dto.TextPIIEntity;
+import com.verygood.security.larky.modules.vgs.cerebro.text.analyzer.spi.TextPIIAnalyzer;
+import net.starlark.java.eval.EvalException;
+import net.starlark.java.eval.Starlark;
+import net.starlark.java.eval.StarlarkFloat;
+
+import java.util.List;
+
```

Comment:
Any reason why this isn't a default interface ?

Reply:
This is a default instance.

Reply:
Ok, I see where the confusion is.
I've renamed it to DefaultTextPIIAnalyzer